image/Dockerfile

```diff
@@ -243,6 +245,7 @@ RUN set -x && export DEBIAN_FRONTEND=noninteractive && \
     (locale -a | tee /dev/stderr | grep -qx en_US.utf8)
 
 # Copy prebuilt items
+COPY --from=prebuilt-setpriv /dist/setpriv /
```

Comment:
I'm not quite following how this is the first thing getting copied out of the "prebuilt-setpriv" intermediate image. You were already compiling a complete copy of util-linux and libcap-ng, but I guess not actively installing anything from them?

Reply:
Right, those are seperate images (as part of a multi-stage build) that get built solely for the purpose of building a static copy of setpriv. Unless one of the final images pulls them in with a line like this, they are unused (and in fact, BuildKit will completely skip building them unless they are required by one of the final target images).

Reply:
I think the thing to note is that none of the images have a "FROM prebuild-setpriv" statement